test/y2network/interface_config_builders/ib_test.rb

```diff
@@ -27,14 +27,12 @@
 
     it "stores ipoib configuration" do
       subject.ipoib_mode = "datagram"
-      subject.save
-      expect(Yast::LanItems.ipoib_mode).to eq "datagram"
+      expect(subject.save).to eq "datagram"
```

Comment:
I would prefert to check the resultant config than the return of the save method